addon/components/base-focusable.js

```diff
@@ -10,6 +10,7 @@ export default Component.extend(EventsMixin, HasBlockMixin, {
   active: false,
   focused: false,
   hover: false,
+  focusable: true,
```

Comment:
Yes same thing can be archived with focusOnlyOnKey - however the intent is completely hidden away.